test/TestSupport.js

```diff
@@ -0,0 +1,22 @@
+

```

Comment:
The indent in this file is a little off. Can you please change it to something consistent with the other files. Thanks!